gui-tk/godley.tcl

```diff
@@ -305,12 +302,11 @@ proc setGodleyDisplay {} {
     foreach c [info commands godleyWindow*.displayStyle] {
         $c $preferences(godleyDisplayStyle)
     }
-    foreach c [info commands godleyIcon.table.multipleEquities] {
+    foreach c [info commands multipleEquities] {
```

Comment:
I think we can assume that the multipleEquities command always exists, so the foreach loop is not necessary

Reply:
Removing this caused the feature to stop working, so I reinstated it. Again, I may not have enough knowldedge of TCL to do this properly.